### ISSUE TITLE
pizza house fire option

### ISSUE THREAD

**Issue by 666threesixes666** [priomsrb/lighttasks#10]
can u install a count down feature that plays a sound / bell / whistle when count down finishes?  (so my pizza does not catch fire along with the house)

**Comment by priomsrb** [comment]
Thanks for filing an issue.

Right now Light Tasks only counts up. Would it be ok if it was possible to set alerts/alarms when the timer reaches a certain value? For example you can set an alert for when the pizza timer reaches 10 mins. Would that be useful for you?

**Comment by 666threesixes666** [comment]
yeah like 150 seconds = popcorn done (right now im trying to get down on 1 hour 40 minutes lasagna done) or 40 hour cumulative for the week stop coding.  gnome timer applet is broken on gentoo, and your sources work fine.

**Comment by 666threesixes666** [comment]
just a hint....  my gnubiff uses "sox" to use a command line "play /usr/src/file.wav" or "play /usr/src/file.mp3"  thanks in advance for the efforts.....
